tea.py

```diff
@@ -0,0 +1,674 @@
+import pybamm
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+class Tea(pybamm.BaseModel):
+    """"""
+
+    def __init__(self, parameter_values=None):
+        self.parameter_values = parameter_values
+        if self.parameter_values is None:
+            print(
+                "Warning: Missing parameter values, please supply parameter values to Tea()"
+            )
+        super().__init__()
+
+    def plot_ocvs(self, parameter_values=None):
```

Comment:
Don't need to pass `parameter_values` here since you get it an an attribute anyway